ui/src/components/common/Header.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Page,
+  PageHeader,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarItem
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { css } from '@patternfly/react-styles';
+import './Header.css';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { toggleHeaderDropdown } from '../../actions/actions-ui';
+
+class Header extends React.Component {
+  onTitleClick = () => {};
+
+  onUserDropdownToggle = () => {
+    this.props.toggleHeaderDropdown();
+  };
+
+  onUserDropdownSelect = () => {
+    this.props.toggleHeaderDropdown();
+  };
+
+  onLogoutUser = () => {
+    console.log('onLogoutUser()');
+  };
+
+  render () {
+    const logoProps = {
+      onClick: () => this.onTitleClick()
+    };
+
+    const toolbar = (
+      <Toolbar>
+        <ToolbarGroup className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnLg)}>
+          <UserIcon />
+          <ToolbarItem className={css(accessibleStyles.screenReader, accessibleStyles.visibleOnMd)}>
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={this.onUserDropdownSelect}
+              isOpen={this.props.isUserDropdownOpen}
```

Comment:
Hey, I was thinking about this one yesterday. Because both methods do the same thing, you could pass the same method to the isOpen for the Dropdown component and to the onToggle of the DropdownToggle component. wdyt?

Reply:
@laurafitzgerald was thinking something similar, to reduce code duplication & make any future changes easier :+1:

Reply:
@damienomurchu This one still needs to be done?

Reply:
@laurafitzgerald misread your previous comment - isOpen is just a boolean value, and onSelect is the event handler.

Reply:
Sorry I mean onSelect and onToggle

Reply:
onToggle opens the dropdown, and onSelect closes it. I think its probably best to keep both methods to keep table implementation as close to standard configuration as possible though.

Reply:
I disagree. We aren't changing the the table implementation, only the methods that are passed to each of the required props. But it's a small detail, as you wish.